pygeos/geometry.py

```diff
@@ -132,7 +132,7 @@ def get_coordinate_dimension(geometry):
 def get_num_coordinates(geometry):
     """Returns the total number of coordinates in a geometry.
 
-    Returns -1 for not-a-geometry values.
+    Returns 0 for not-a-geometry values.
```

Comment:
"not-a-geometry" is not a term we are using in pygeos, mostly we talk about 'missing' or just "None"  values.

Reply:
The only change here was the return value; `not-a-geometry` was there originally.  For clarity, perhaps we should address changing that in a different PR?